cdap-docs/developers-manual/source/building-blocks/applications.rst

```diff
@@ -10,16 +10,16 @@ Applications
 
 .. highlight:: java
 
-An **Application** is a collection of application virtualizations that read from—and write
-to—the data virtualization layer in CDAP. 
+An **Application** is a collection of building blocks that read and write data through the data
+abstraction layer in CDAP. 
 
-**Application virtualizations** include *Programs,* *Services,* and *Procedures.*
+**Applications** are composed from *Programs,* *Services,* and *Procedures (deprecated).*
```

Comment:
I still think that we should not mention Procedures here. If it seems too little without, perhaps say, from Programs, Services, and Schedules.